src/modules/auth/actions/update-is-logged.js

```diff
@@ -0,0 +1,10 @@
+export const UPDATE_IS_LOGGED = 'UPDATE_IS_LOGGED';
+
+export const updateIsLogged = isLogged => (
```

Comment:
Suggest renaming this to `updateIsLoggedIn`; I think `isLogged` is a bit ambiguous.